modules/heat_transfer/src/materials/ElectromagneticHeatingMaterial.C

```diff
@@ -0,0 +1,110 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#include "ElectromagneticHeatingMaterial.h"
+#include "FormulationEnums.h"
+
+registerMooseObject("HeatTransferApp", ElectromagneticHeatingMaterial);
+
+InputParameters
+ElectromagneticHeatingMaterial::validParams()
+{
+  InputParameters params = ADMaterial::validParams();
+  params.addClassDescription(
+      "Material class used to provide the electric field as a material property and computes the "
+      "residuals for electromagnetic/electrostatic heating based objects.");
+  params.addCoupledVar(
+      "field", "The electic field vector or electrostatic potential scalar to produce the field.");
+  params.addCoupledVar(
+      "complex_field",
+      "The complex component of the electic field vector for the harmonic formulation.");
+  params.addParam<std::string>(
+      "field_material_name", "field", "User-specified material property name for the field.");
+  params.addParam<std::string>("field_heating_name",
+                               "field_heating",
+                               "User-specified material property name for the Joule Heating.");
+  params.addParam<Real>("heating_scaling", 1.0, "Coefficient to multiply by heating term.");
+  params.addParam<MaterialPropertyName>(
+      "conductivity",
+      "conductivity",
```

Comment:
I'd strongly recommend naming this parameter electrical_conductivity, both in the Params (for input file use) and for in the code. Since this material lives in the heat transfer module, the default assumption for conductivity will be thermal conductivity, not electrical. A few more letters in the name will help remove that ambiguity

Reply:
That's fair enough. My default thinking was electrical conductivity due to it being electromagnetic based heating, but we should reduce ambiguity when possible. I made the edit to `electrical_conductivity`.